components/notebook-controller/go.mod

```diff
@@ -0,0 +1,73 @@
+module github.com/kubeflow/kubeflow/components/notebook-crd
```

Comment:
this should be changed to notebook-controller so it agrees with the new path

Reply:
done